duckduckgo/Android
# Blank web content if user is redirected to another site and new content does not load

Task/Issue URL: https://app.asana.com/0/488551667048375/1196139873349464/f
Tech Design URL: https://app.asana.com/0/481882893211075/1195526928832633/f
CC: 

**Description**:
This PR fixes an issue of updating the URL while showing previous content during more time than expected.

How to reproduce:
1. Visit https://duckduckgo.github.io/privacy-test-pages/security/spoof-js-page-rewrite.html
2. Press run
3. click on "start"
4. Ensure user is redirected to a new site, and loading freezes with wrong content while showing duck.co in the omnibar

It could happen that genuine website (duck.co) loads and fake content does not appear. Try again after clearing all data (with fire button) to reproduce the issue (maybe you will need to do it multiple times, not very deterministic).

**Steps to test this PR**:
#Test 1: 
1. Open DDG app (and press fire to be sure we clear the cache)
1. Visit https://www.w3schools.com/code/tryit.asp?filename=GJA5PVPWMMIJ
2. Press run
3. click on "load and redirect"
4. Ensure url is updated to duck.co and fake content is visible for 1 second
5. Ensure web content hides after 1 second
6. Wait 10 seconds
7. Ensure new content appears

#Test 2:
Test the app browsing as you usually do on your daily basis:
Try searching something at DDG and click on some links.
Try to log in on any website (just to be redirected to another site automatically)
Ensure user experience is good as a user.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
